### PR TITLE
test: Add basic postgres database image test using pgTAP

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmi@supabase.io>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
tested locally and introduced errors. Build fails with error like:  
err

File: ebssurrogate/files/unit-tests/unit-test-01.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+-- Check roles
+SELECT has_role('anon');
+SELECT has_role('dashboard_user');
+SELECT has_role('service_role');
+SELECT has_role('supabase_admin');
+SELECT has_role('supabase_auth_admin');
+SELECT has_role('supabase_storage_admin');
+
+-- Check auth schema relations
+SELECT fk_ok( 'auth', 'identities',   'user_id', 'auth',  'users', 'id');
```
Comment (darora) [inline ebssurrogate/files/unit-tests/unit-test-01.sql]: what's with the odd spacing between the params? intentional?

File: ansible/playbook.yml [hunk 2]
```diff
@@ -150,6 +153,23 @@
       shell:
         cmd: "for fl in /usr/lib/postgresql/bin/* ; do ln -sf $fl /usr/bin/$(basename $fl) ; done"
 
+    - name: Enable pgTAP extension
```
Comment (darora) [inline ansible/playbook.yml]: let's move these out to a separate task file that gets included

File: ansible/playbook.yml [hunk 3]
```diff
@@ -150,6 +153,23 @@
...
+      become: yes
```
Comment (darora) [inline ansible/playbook.yml]: `become: yes` should be unnecessary given it's configured for the entire playbook?

**Review by darora — APPROVED** [review]

File: ansible/playbook.yml (hunk 2 above)
Comment (Lakshmipathi) [reply ansible/playbook.yml]: okay, will update the PR

File: ebssurrogate/files/unit-tests/unit-test-01.sql (hunk 1 above)
Comment (Lakshmipathi) [reply ebssurrogate/files/unit-tests/unit-test-01.sql]: fixed it.

**Review by pcnc — APPROVED** [review]

**Pull request merged**